scormxblock/scormxblock.py

```diff
@@ -424,7 +432,17 @@ def get_raw_scorm_status(self, request, suffix=''):
         """
         # TODO: handle errors
         # TODO: this is specific to SSLA player at this point.  evaluate for broader use case
-        return Response(self.raw_scorm_status, content_type='application/json', charset='UTF-8')
+        response = Response(self.raw_scorm_status, content_type='application/json', charset='UTF-8')
+        self.update_progress(response.body)
```

Comment:
Please use a more descriptive name like `publish_complete_progress_if_required(...)`
You can also rename it to `publish_complete_progress(...)` and bring the if condition out of the function into get_raw_scorm_status()